src/test/scala/reporter/CloudWatchReporterSpec.scala

```diff
@@ -0,0 +1,35 @@
+package reporter
+
+import com.blacklocus.metrics.CloudWatchReporterBuilder
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{FlatSpec, Matchers, PrivateMethodTester}
+import Utils.buildMetricFilter
+import scala.util.matching.Regex
+
+
+class CloudWatchReporterSpec extends FlatSpec with Matchers with PrivateMethodTester with MockFactory {
```

Comment:
This class doesn't match the object that is containing the code (which is currently `Utils`)